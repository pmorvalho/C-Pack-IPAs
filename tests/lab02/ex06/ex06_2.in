3 -1.8 3.14 1