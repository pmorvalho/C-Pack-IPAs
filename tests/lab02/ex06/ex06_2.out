min: -1.800000, max: 3.140000
